### PR TITLE
[chore] remove jpkrohling as codeowner from many modules

### PR DESCRIPTION
**Pull request by jpkrohling**
This PR removes @jpkrohling as code owner for multiple modules.

Signed-off-by: Juraci Paixão Kröhling <juraci@kroehling.de>

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -233,7 +233,7 @@ receiver/googlecloudspannerreceiver/              @open-telemetry/collector-cont
 receiver/haproxyreceiver/                         @open-telemetry/collector-contrib-approvers @atoulme @MovieStoreGuy
 receiver/hostmetricsreceiver/                     @open-telemetry/collector-contrib-approvers @dmitryax @braydonk
 receiver/httpcheckreceiver/                       @open-telemetry/collector-contrib-approvers @codeboten
-receiver/huaweicloudcesreceiver/                  @open-telemetry/collector-contrib-approvers @narcis96
+receiver/huaweicloudcesreceiver/                  @open-telemetry/collector-contrib-approvers @heitorganzeli @narcis96 @mwear
```
Comment (jpkrohling) [inline .github/CODEOWNERS:236]: I'm not sure how this got in, but #36258 seems to have introduced all three code owners, but this file got only one. Generating the code owners file again appears to have caught this difference.

File: cmd/githubgen/allowlist.txt [hunk 2]
```diff
@@ -1,23 +1,24 @@
+abhishek-at-cloudwerx
+adcharre
 Caleb-Hurshman
+cemdk
 cheempz
-jerrytfleung
```
Comment (jpkrohling) [inline cmd/githubgen/allowlist.txt:3]: I sorted this file alphabetically.

Comment (marctc) [reply cmd/githubgen/allowlist.txt:3]: I'm not in this list, could you add me if required?

Comment (jpkrohling) [reply cmd/githubgen/allowlist.txt:3]: You are a member of the org, this file lists people who are code owners but NOT part of the OTel organization.

**Review by mx-psi — COMMENTED** [review]
The following components will be unmaintained (no code owners) after this change:
- headersetter extension
- healtcheck extension
- awscloudwatchmetrics receiver

Should we mark them as unmaintained in this PR? I feel like the healthcheck extension may find new code owners easily if we ask around.

Also, no need to deal with these on this PR, but the following components will be unmaintained after https://github.com/open-telemetry/opentelemetry-collector/pull/11616 is merged:
- exceptions connector
- loki receiver

At a minimum we should mark them as `seeking_new: true`

**Comment by jpkrohling** [comment]
> headersetter extension

I'll add the seek_new to this.

> healtcheck extension

This will eventually be replaced by the v2 (cc @mwear)

> awscloudwatchmetrics receiver

This is in development and abandoned by the original author. I would just remove the code we have there, as there are no users of it at the moment. I was planning on doing that on a follow-up PR. 

> exceptions connector

I don't think this is a vendor-specific component. cc @marctc 

> loki receiver

I believe @mar4uk is still around for that one.

**Comment by marctc** [comment]
> > exceptions connector
> 
> I don't think this is a vendor-specific component. cc @marctc

No, is not.

**Review by mx-psi — APPROVED** [review]
Okay, I think I misremembered https://github.com/open-telemetry/opentelemetry-collector/pull/11616 :sweat_smile: I think the only ones we care about are the ones that have no codeowners now. 

I agree with your assessment, but I think we should mark them as unmaintained to follow process. We can wait for a few days in case somebody shows up and mark as unmaintained if not

**Review by codeboten — APPROVED** [review]

**Comment by mx-psi** [comment]
Filed #36598 and #36599

**Pull request merged**